test/SlowTests/Issues/RavenDB-19205.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Linq;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_19205 : RavenTestBase
+{
+    public RavenDB_19205(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    //https://github.com/dotnet/runtime/pull/67666
+    [Fact]
+    public void DatePrecisionTest()
```

Comment:
can we move this test to EmbeddedTests? This way you will have `net7.0;net6.0;net5.0;netcoreapp3.1;net462` (client API) vs `net7` (server) coverage

Reply:
done